### PR TITLE
limit chunking to integer size (master)

### PR DESCRIPTION
**Pull request by d-w-moore**

### CODE REVIEW

**Comment by trel** [comment]
tiny codacy, obviously not a blocker

**Comment by d-w-moore** [comment]
Took care of codacy.  Now running a final test on ubuntu_16.  Likely we can # and merge in the morning

File: server/api/src/rsDataObjRead.cpp [hunk 1]
```diff
@@ -15,6 +15,8 @@
 #include "rsDataObjLseek.hpp"
 
 #include <algorithm>
+#include <type_traits>
```
Comment (korydraughn) [inline server/api/src/rsDataObjRead.cpp:18]: Is the `<type_traits>` header still needed?

**Comment by d-w-moore** [comment]
force push, #'d , took care of extra include and codacy concerns

**Pull request merged**